.docker/fedora/Dockerfile

```diff
@@ -0,0 +1,59 @@
+ARG BASE
+FROM ghcr.io/dschwoerer/bout-container-base:$BASE
+
+# ----------------------------------------------------------------
+# Build and install BOUT++
+# ----------------------------------------------------------------
+# password: boutforever
+RUN useradd boutuser -G wheel -p '$6$MoHfQDiMU5ajgDMm$9FAMLxMflKwQCZ.sJBNG6wLGnPeySVizdA8wN0k8LSXKPkCfOb/sM9Y4jKFvh5rzKSwLtYSTzvJyETqrFlxBV.'
+
+RUN echo '%wheel ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+USER boutuser
+WORKDIR /home/boutuser
+
+ARG MPI
+ARG URL
+ARG COMMIT
+ARG CMAKE_OPTIONS
+
+RUN sudo ls
+
+# Checkout submodules now so configure later is fast, and iterating on
+# it less painful
+RUN git clone $URL \
+ && chown -R boutuser /home/boutuser/BOUT-dev \
+ && cd BOUT-dev \
+ && git checkout $COMMIT \
+ && git -c submodule.recurse=false submodule update --init --recursive
```

Comment:
This looks a bit curious -- does `--recursive` not imply `submodule.recurse=true`?

Reply:
Yes, that _should_ be fine, but as you point out here, might be faster to do that first?

Reply:
Oh, I missed that comment :+1: